content/news/this-week-in-fluvio-0054.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "This Week in Fluvio #54"
+date: 2023-07-11
+weight: 20
+---
+Fluvio is a distributed, programmable streaming platform written in Rust.
+
+---
+Alright, alright, alright! we are back on time for this week in Fluvio - 54.
+
+This week hase been great with our efforts aligning towards our next release.
+
+## Upcoming features
+We have updates coming in the interface of the [SQL sink connector]('https://github.com/infinyon/sql-connector') that will provide the option to configure insert and upsert operations that we are about to release.
+
+We spent some time working on a graphite connector which we will make available through [InfinyOn Labs]('https://github.com/infinyon/labs-projects')
+
+## Developer experience improvements
+We are also looking at the Fluvio [Python]('https://github.com/infinyon/fluvio-client-python') and [Go]('https://github.com/avinassh/fluvio-go') clients to get a baseline of functionality.
+
+We are experimenting with how far we can push our core stream processor in terms of efficiency under significant storage and memory constraints.
+
+## New blog post
+[How to use SQL Sink Connector Upsert]('https://infinyon.com/blog/2023/07/sql-upsert/')
+
+## New video
+[Collecting Hackernews RSS Feeds using InfinyOn Cloud]('https://youtu.be/fVzLoaIHQfM')
+
+That's all folks. until next week.
+
+---
```

Comment:
Let's add `We are hiring` - and a link to the position
https://infinyon.com/careers/cloud-engineer-senior-level/

Reply:
Done